gulpfile.js

```diff
@@ -5,57 +5,97 @@
 const awspublish = require('gulp-awspublish');
 const babel = require('gulp-babel');
 const gulp = require('gulp');
+const path = require('path');
 const rename = require('gulp-rename');
 const template = require('gulp-template');
 const uglify = require('gulp-uglify');
+const umd = require('gulp-umd');
 const {version} = require('./package');
 
-// building
+/**
+ * UMD wrap sdk.js
+ */
+gulp.task('build:umd', function() {
+  return gulp.src('src/sdk.js')
+    .pipe(umd({
+      // CommonJS export name
+      exports: function() { return 'Smartcar'; },
+      // Global namespace for Web.
+      namespace: function() { return 'Smartcar'; },
+      // returnExports template with istanbul ignore
+      template: path.join(__dirname, 'build/returnExports.js'),
+    }))
+    .pipe(gulp.dest('dist/umd'));
+});
 
-// builds js files by babeling, uglifying, and versioning
-gulp.task('build-js', function() {
+/**
+ * Build sdk.js for npm publishing.
+ */
+gulp.task('build:npm', ['build:umd'], function() {
+  return gulp.src('src/sdk.js')
+    .pipe(babel())
+    .pipe(gulp.dest('dist/npm'));
+});
+
+/**
+ * Build JS for CDN publishing.
+ */
+gulp.task('build:cdn:js', ['build:umd'], function() {
   return gulp.src('src/*.js')
     .pipe(babel())
     .pipe(uglify())
     .pipe(rename({suffix: `-${version}`}))
-    .pipe(gulp.dest('dist'));
+    .pipe(gulp.dest('dist/cdn'));
 });
 
-// builds html file by templating and versioning
-gulp.task('build-html', function() {
+/**
+ * Build HTML for CDN publishing
+ */
+gulp.task('build:cdn:html', function() {
   return gulp.src('src/redirect.html')
     .pipe(template({redirectJS: `'/redirect-${version}.js'`}))
```

Comment:
nit: single quotes in HTML

Reply:
:+1: i'm moving the string into the HTML file and only passing the version through the template dat